pkg/antctl/transform/addressgroup/transform.go

```diff
@@ -30,11 +35,30 @@ type Response struct {
 }
 
 func listTransform(l interface{}, opts map[string]string) (interface{}, error) {
-	groups := l.(*cpv1beta.AddressGroupList)
-	result := []interface{}{}
-	for i := range groups.Items {
-		item := groups.Items[i]
-		o, _ := objectTransform(&item, opts)
+	groupsList := l.(*cpv1beta.AddressGroupList)
+	sortField := opts["sort-by"]
+	if sortField == "" {
+		sortField = ".metadata.name"
+	}
+
+	var addressGroupRuntime runtime.Object = &cpv1beta.AddressGroupList{
+		Items: groupsList.Items,
+	}
+
+	var addressGroupRuntimeObjectList []runtime.Object
+	meta.EachListItem(addressGroupRuntime, func(item runtime.Object) error {
+		addressGroupRuntimeObjectList = append(addressGroupRuntimeObjectList, item)
+		return nil
+	})
```

Comment:
we can't use https://pkg.go.dev/k8s.io/apimachinery/pkg/api/meta#ExtractList here?